.github/workflows/check-catalyst.yaml

```diff
@@ -455,7 +455,9 @@ jobs:
         fail_ci_if_error: true
         token: ${{ secrets.CODECOV_TOKEN }}
 
+    # TODO: Find fix for demos with gcc 
     - name: Check Catalyst Demos
+      if: ${{ matrix.compiler != 'gcc' }}
```

Comment:
Just curious when was this discovered? :)

Reply:
Today at merging RC int main

Reply:
Well, it has been going on for a while, there's a discussion on slack. I'll tag you.

Reply:
Does it have anything to with gcc? It's the only configuration right now no?

Reply:
Yes, it is the only one

Reply:
Let's make it clear that this is a general skip, rather than anything specific to gcc then. Let's also make sure there is a bug report on shortcut for it :)